### PR TITLE
Text changes

### PR DESCRIPTION
**Pull request by defnotryan**
List of required changes on Trello: https://trello.com/c/ot0vRxlz/38-text-changes

I may add another commit to this if it turns out Eric has a bundle for the Frank font.

### CODE REVIEW

**Comment by defnotryan** [comment]
memorize_ephesians_1-3

**Comment by defnotryan** [comment]
memorize_ephesians_1-3

**Comment by calebwoods** [comment]
Only thought is whether we want to keep Helvetica / Source Code Pro for the navigation elements of the  page (buttons, arrows, etc).  They look a like a strange as Serif font.

**Comment by calebwoods** [comment]
I also merged the other so you can resolve the conflicts with this change

**Comment by defnotryan** [comment]
@calebwoods I replaced the nav arrows (previously `&gt;` and `&lt;`) with angle arrows from Font Awesome; that helps a lot. However I wasn't sure what you meant by the other navigation elements -- these are sans-serif (Frank) and downgrade according to the `$ui-font-stack` defined in `_variables.css`. Is there something in this patch I'm missing that would make them downgrade to a serif font?

File: css/base/_base.css [hunk 1]
```diff
@@ -19,30 +19,25 @@ html {
...
 
 .wrapper {
-	color: #222222;
-	font-family: monospace;
-	max-width: $large-width;
-	margin: 0 auto;
-	white-space: pre-wrap;
-}
-
-body.js-source-code-pro-loaded .wrapper {
```
Comment (calebwoods) [inline css/base/_base.css:39]: If we are removing Source Code Pro completely there was some logic in app.js that can now be removed.

**Comment by calebwoods** [comment]
@defnotryan I see what you mean now about the Frank font.  I was mainly looking at the next/previous buttons.  Just added a comment about removing checks for the Source Code Pro

**Comment by defnotryan** [comment]
Ok, so the font bundle Eric posted on Trello doesn't contain anything I didn't already have, so no commits there. (I.e., leaving it as OTF).

Also, I removed the rest of the Source Code Pro stuff from `app.js`.

**Comment by defnotryan** [comment]
Any objections to merging?

**Comment by calebwoods** [comment]
Merge away

**Pull request merged**